runtime/src/accounts_db.rs

```diff
@@ -3254,7 +3257,12 @@ impl AccountsDb {
             true
         } else {
             false
-        }
+        };
+        info!(
```

Comment:
I think it would also be good to log:

1)  the `requested_flush_root` and `max_clean_root` in  `flush_rooted_accounts_cache`

2) Both `max_clean_root` in `clean_accounts()` here: https://github.com/solana-labs/solana/blob/master/runtime/src/accounts_db.rs#L1376-L1377